packages/injected/src/wallets.ts

```diff
@@ -78,31 +83,22 @@ const binance: InjectedWalletModule = {
     window.BinanceChain.on = (event, func) => {
       // intercept chainChanged event and format string
       if (event === 'chainChanged') {
-        addListener(event, (chainId: string) => {
+        addListener(event, (chainId: ChainId) => {
           const cb = func as ChainListener
-          cb(`0x${parseInt(chainId).toString(16)}`)
+          cb(`0x${parseInt(chainId as string).toString(16)}`)
         })
       } else {
         addListener(event, func)
       }
     }
 
     const provider = createEIP1193Provider(window.BinanceChain, {
-      // If the wallet is unlocked then we don't need to patch this request
-      ...(!window.BinanceChain.isUnlocked && {
-        eth_accounts: () => Promise.resolve([])
-      }),
-      eth_requestAccounts: ({ baseRequest }) =>
-        baseRequest({ method: 'eth_requestAccounts' }).then(accts => {
-          window.BinanceChain.isUnlocked = true
-          return accts
-        }),
-      eth_selectAccounts: UNSUPPORTED_METHOD,
       eth_chainId: ({ baseRequest }) =>
-        baseRequest({ method: 'eth_chainId' }).then(
-          id => `0x${parseInt(id).toString(16)}`
-        ),
+        baseRequest({ method: 'eth_chainId' }).then(id => {
+          return `0x${parseInt(id as string).toString(16)}`
+        }),
```

Comment:
```suggestion
        baseRequest({ method: 'eth_chainId' }).then(id => `0x${parseInt(id as string).toString(16)}`),
```

Reply:
Good catch! Thanks! Fixed on branch for formatting